modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -221,7 +221,8 @@ TEST_P(Test_ONNX_layers, Deconvolution)
     testONNXModels("two_deconvolution", npy, 0, 0, false, false);
     testONNXModels("deconvolution_group", npy, 0, 0, false, false);
     testONNXModels("deconvolution_output_shape", npy, 0, 0, false, false);
-    testONNXModels("deconv_adjpad_2d", npy, 0, 0, false, false);
+    if (target != DNN_TARGET_CUDA_FP16) // bug
+        testONNXModels("deconv_adjpad_2d", npy, 0, 0, false, false);
```

Comment:
This test case has five tests. Only the last test fails for CUDA FP16 target. I haven't skipped since the other tests are still useful but not skipping might make it harder to track (and enable it in the future after the bug is fixed).

Maybe it would be a good idea to split this tests into multiple tests (or at least split the one that is failing into its own test).

Reply:
> split this tests into multiple

Agreed.

/cc @l-bat @dkurt @sl-sergei